phalcon/assets/manager.zep

```diff
@@ -488,6 +488,16 @@ class Manager
 					let prefixedPath = path;
 				}
 
+                let version = collection->getVersion();
+				if $resource->getVersion() == null && version != null {
+					if version {
+						if version === true && local {
+							let version = filemtime($resource->getRealSourcePath());
+						}
+						let prefixedPath = prefixedPath . "?ver=" . version;
+					}
+				}
```

Comment:
Ideologically the version must be a string or null. Not boolean or yet another type.